base/docs/helpdb/Base.jl

```diff
@@ -130,13 +130,28 @@ Obtain the contents of an `IOBuffer` as an array, without copying. Afterwards, t
 takebuf_array
 
 """
-    download(url,[localfile])
+    download(url [localfile] [silent=false])
```

Comment:
Ah, and since this is a keyword argument, you need to put `;` before `silent` (instead of a comma, contrary to what I said above).